src/pages/ReportParticipantsPage.js

```diff
@@ -101,7 +101,10 @@ function ReportParticipantsPage(props) {
     return (
         <ScreenWrapper includeSafeAreaPaddingBottom={false}>
             {({safeAreaPaddingBottomStyle}) => (
-                <FullPageNotFoundView shouldShow={_.isEmpty(props.report) || ReportUtils.isArchivedRoom(props.report)}>
+                <FullPageNotFoundView
+                    shouldShow={_.isEmpty(props.report) || ReportUtils.isArchivedRoom(props.report)}
+                    onBackButtonPress={() => Navigation.goBack(ROUTES.getReportRoute(props.report.reportID))}
```

Comment:
Why is this change needed?

Reply:
This should go to Report Details page
```suggestion
                    onBackButtonPress={() => Navigation.goBack(ROUTES.getReportDetailsRoute(report.reportID))}
```